docs/getting-started/documentation/api_wrapper.md

```diff
@@ -0,0 +1,34 @@
+Manging the EDC Connector via the API Wrapper Java Client Library
+========
+
+Introduction to the sovity EDC API Wrapper
+========
+The sovity EDC API Wrapper contains several APIs that are shipped with the Connector-as-a-Service (CaaS) sovity EDC Connectors and are made accessible via the type-safe generated client libraries. Please note that most of these APIs are not yet complete and are under development:
```

Comment:
contains several APIs, of which some are available in either our sovity EDC CE or our sovity CE EE / Connector-as-a-Servcie (CaaS). These APIs are made accessible via type-safe generated client libraries.